pur/__init__.py

```diff
@@ -310,6 +310,18 @@ def _update_requirements(obuffer, updates, input_file=None,
                 if not dry_run_changed:
                     obuffer.write(line)
 
+        elif not latest_ver and not output_buffer:
+            if not dry_run_changed:
+                msg = '\033[91mCould not find a version that satisfies the ' \
+                      'requirement {package} (from -r req.txt (line 1))' \
+                      ' (from versions: {version})\033[0m'.format(
+                    package=req.name,
+                    version=latest_ver
+                    )
+                if echo and not dry_run:
```

Comment:
Shouldn't we show the warning also in dry runs `--dry`?

Reply:
Sounds good, I'll add that